### PR TITLE
Add Readme

### PR DESCRIPTION
**Pull request by rydrman**
adds readme file with some basic instructions
updates go files to use 4-space tabs

### CODE REVIEW

**Comment by rydrman** [comment]
hound seems to be working now... there's no real output information so maybe it was just a bad branch or something

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# ktrlpy
+
+[![Build Status](https://travis-ci.org/KloudKtrl/ktrlpy.svg?branch=master)](https://travis-ci.org/KloudKtrl/ktrlpy) [![Coverage Status](https://coveralls.io/repos/github/KloudKtrl/ktrlpy/badge.svg)](https://coveralls.io/github/KloudKtrl/ktrlpy)
+
+ktrlpy is a python library for ktrl.io
+
+## Installation
+
+Currently there are a few steps to getting this library setup for use in pyhton.
```
Comment (seanhagen) [inline README.md]: pyhton -> python

**Comment by seanhagen** [comment]
Something for further on down the line: it'd be nice if the Python install script could just download the right shared object ( the compiled C bit ) for the platform instead of users having to install Go. Something we should investigate after the Proof of Concept is done.

**Comment by rydrman** [comment]
ya python has a set of setup tools and a package repo that we should support in the future. I was going to do it now but it's all build around compiling C, so it's gonna be a bit of a struggle to get it compiling go, I'll add it to the backlog

**Pull request merged**